wagtailio/services/templates/services/blocks/card.html

```diff
@@ -0,0 +1,16 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+{% with value as card %}
+<div class="card">
+    <div class="card__upper">
+        <div>
```

Comment:
nitpick: is this `<div>` needed?

Reply:
True, that must be a leftover. Removed in 07880e9